### PR TITLE
Add little hug_formatter for messages with concrete realization.

### PR DESCRIPTION
**Pull request by krosenmann**
- Make sended text by `/me` __italic__.
- Add **bold** analogue for future use.

### CODE REVIEW

**Comment by krosenmann** [comment]
@Cuttlerat testy jdyom

**Comment by kosc** [comment]
@Cuttlerat why there is still one review required?

**Comment by krosenmann** [comment]
> @Cuttlerat why there is still one review required?

Coz we are waiting for tests of `/me`. I guess.

File: bot/utils.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from functools import partial
+
+
+def _hug_text(text, formatter):
+    return f'{formatter}{text}{formatter}'
+
+
+italize = partial(_hug_text, formatter='_')
+boldize = partial(_hug_text, formatter='*')
```
Comment (Cuttlerat) [inline bot/utils.py]: Unused, please remove

**Comment by Cuttlerat** [comment]
> > @Cuttlerat why there is still one review required?
> 
> Coz we are waiting for tests of `/me`. I guess.

I've created the #74 issue for this, as it's not related to this PR

**Pull request merged**